Google.Cloud.Spanner.NHibernate/Google.Cloud.Spanner.NHibernate.csproj

```diff
@@ -2,11 +2,22 @@
 
     <PropertyGroup>
         <TargetFramework>netstandard2.0</TargetFramework>
+        <AssemblyName>Google.Cloud.Spanner.NHibernate</AssemblyName>
+        <RootNamespace>Google.Cloud.Spanner.NHibernate</RootNamespace>
         <LangVersion>latest</LangVersion>
+        <Authors>Google LLC</Authors>
+        <Copyright>Copyright 2022 Google LLC</Copyright>
+        <PackageLicenseFile>LICENSE</PackageLicenseFile>
+        <PackageProjectUrl>https://github.com/googleapis/dotnet-spanner-nhibernate</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/googleapis/dotnet-spanner-nhibernate</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <PackageTags>spanner;nhibernate;Google;Cloud</PackageTags>
+        <PackageIcon>NuGetIcon.png</PackageIcon>
```

Comment:
I think we had to add the icon URL as it was not picked otherwise.

Reply:
Thanks for the tip. I've added that to this project as well.